stix2/test/test_observed_data.py

```diff
@@ -761,14 +761,13 @@ def test_file_example_with_WindowsPEBinaryExt():
 
 
 def test_file_example_encryption_error():
-    with pytest.raises(stix2.exceptions.DependentPropertiesError) as excinfo:
-        stix2.File(name="qwerty.dll",
-                   is_encrypted=False,
-                   encryption_algorithm="AES128-CBC"
-                   )
+    file_ = stix2.File(name="qwerty.dll", is_encrypted=False,
```

Comment:
You are correct. I think I misunderstood what we were testing here. Yes, this is brought up by the changes to `getattr`.